assignments/assignment-6/public/scripts.js

```diff
@@ -214,3 +281,75 @@ $(document).ready(function() {
     }
   });
 });
+
+function getQuestion(category, pointVal){
+    $.ajax({
+     url: "/jeopardy/" + category + "/" + pointVal,
+    success: function(data) {
+      console.log(data);
+      displayQuestion(data);
+    },
+    error: function(err) {
+      console.log('Error retrieving question: ' + JSON.stringify(err));
+   }
+  });
+}
+
+function displayQuestion(data){
+    currentScoreAmount = parseInt(data[0].pointValue);
+    pointsClicked = false;
```

Comment:
What is this doing?

Reply:
oh I see you set up a global var that is acting as a flag to determine when to add points...